airflow-core/tests/unit/api_fastapi/core_api/routes/public/test_task_instances.py

```diff
@@ -2312,10 +2311,103 @@ def test_dag_run_with_future_or_past_flag_returns_400(self, test_client, session
         response = test_client.post(f"/dags/{dag_id}/clearTaskInstances", json=payload)
         assert response.status_code == 400
         assert (
-            "Cannot use include_past or include_future when dag_run_id is provided"
+            "Cannot use include_past or include_future with a manually triggered DAG run (logical_date is None)."
             in response.json()["detail"]
         )
 
+    @pytest.mark.parametrize(
+        "flag, expected",
+        [
+            ("include_past", 2),  # D0 ~ D1
+            ("include_future", 2),  # D1 ~
```

Comment:
```suggestion
            ("include_future", 2),  # D1 ~ D2
```

Maybe change `DX` to `TX`, T1, T2, T3, I don't understand why the Task abervation ends up being a `D`

Reply:
applied it